x/leverage/keeper/keeper_test.go

```diff
@@ -11,6 +11,7 @@ import (
 	tmrand "github.com/tendermint/tendermint/libs/rand"
 	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
 
+	"github.com/umee-network/umee/app"
```

Comment:
```suggestion
```

In tests, use `s.app` from the rest suit - avoid importing the umee app itself.

Reply:
done at https://github.com/umee-network/umee/pull/392/commits/29fb07ac3301b97bab5c3941ae65e064f4efbb38